app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MediaRanker</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Gudea:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+  <header class="app-header__header">
+    <h1 id="media">
+      <%= link_to "Media Ranker", root_path%>
+    <small>Ranking the Best of Everything</small>
+    </h1>
+
+    <nav class="app-header__nav">
+      <ul>
+        <li class="list-inline-item"><%= link_to "View top media", root_path%></li>
+        <li class="list-inline-item"><%= link_to "View all media", works_path%></li>
+        <li class="list-inline-item"><%= link_to "Add a new work", new_work_path%></li>
+        <li class="list-inline-item"><%= link_to "View all users", users_path %></li>
+      </ul>
+      <ul>
+        <% if session[:user_id] %>
+<!--          it's a link not a button!-->
+        <li class="header_button"><%= link_to "Logout", logout_path, method: :post, class: "btn btn-info" %></li>
+          <% else %>
+        <li class="header_button"><a href="/login"><button type="button" class="btn btn-info">Log in</button></a></li>
+          <% end %>
+      </ul>
+    </nav>
+  </header>
+
+<!--  <section class="flash-msg__section">-->
+    <%# flash.each do |name, message| %>
+<!--      <strong class="<%#= name %>">-->
+        <%#= message %>
+<!--      </strong>-->
+    <%# end %>
+<!--  </section>-->
+
+  <%if !!flash[:error] %>
+    <section class="alert alert-danger">
+      <%= flash[:error] %>
+    </section>
+  <%end %>
+
+  <%if !!flash[:notice] %>
+    <section class="alert alert-warning">
+      <%= flash[:notice] %>
+    </section>
+  <%end %>
+
+  <%if !!flash[:success] %>
+    <section class="alert alert-success">
+      <%= flash[:success] %>
+    </section>
+  <%end %>
```

Comment:
This can be cleaned up a bit.   Consider using a loop through flash and using the keys for class names in styling.